package.json

```diff
@@ -36,14 +40,17 @@
     "testEnvironment": "node"
   },
   "dependencies": {
-    "is-any-array": "^2.0.0"
+    "is-any-array": "^2.0.1"
   },
   "devDependencies": {
-    "@babel/plugin-transform-modules-commonjs": "^7.16.8",
-    "eslint": "^8.10.0",
-    "eslint-config-cheminfo": "^7.2.2",
-    "jest": "^27.5.1",
-    "prettier": "^2.5.1",
-    "rollup": "^2.68.0"
+    "@types/jest": "^29.5.3",
+    "@vitest/coverage-v8": "^0.34.1",
+    "cheminfo-types": "^1.7.2",
```

Comment:
You're using this package in the published types. It must be in prod dependencies